apps/kyc-server/src/components/ErrorBoundary.tsx

```
@@ -0,0 +1,56 @@
+					<div className="flex flex-col items-center justify-center min-h-[400px] p-8">
+						<div className="text-center">
+							<h2 className="text-xl font-semibold text-foreground mb-2">
+								Something went wrong
+							</h2>
+							<p className="text-muted-foreground mb-4">
+								Failed to load the page. Please try refreshing.
+							</p>
+							<button
+								onClick={() => window.location.reload()}
+								className="px-4 py-2 bg-primary text-primary-foreground rounded-md hover:bg-primary/90 transition-colors"
+							>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)